src/main/java/io/odpf/firehose/metrics/Instrumentation.java

```diff
@@ -17,7 +19,6 @@
 import static io.odpf.firehose.metrics.Metrics.GLOBAL_MESSAGES_TOTAL;
 import static io.odpf.firehose.metrics.Metrics.MESSAGE_SCOPE_TAG;
 import static io.odpf.firehose.metrics.Metrics.MESSAGE_TYPE_TAG;
-import static io.odpf.firehose.metrics.Metrics.MessageType;
```

Comment:
keep this and don't change the method signature.

Reply:
reverted